.github/workflows/update-image.yml

```diff
@@ -8,9 +8,6 @@ on:
       - '**'
     paths:
       - Dockerfile
-      - .dockerignore
-      - Pipfile
-      - Pipfile.lock
```

Comment:
We should add `requirements.txt` here to ensure that the ncov-ingest image gets updated whenever we update dependencies.

Reply:
Thanks for adding this to #439!